chapters/tutorials/tutorial_01.rst

```diff
@@ -0,0 +1,145 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+What is Jina
+----------------------------------
+
+Tutorial
+----------------------------------
+
+At the end of this tutorial, you will have your search engine using text. You will use text as an input and get a matching text for that.
+For this example, we will use the `Kaggle Wikipedia Corpus <https://www.kaggle.com/mikeortman/wikipedia-sentences>`_
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
+----------------------------------
+
+The first part is to install Jina.
+
+  .. code-block:: python
+
+    pip install jina
+
+And we will also need the following dependencies:
+
+  .. code-block:: python
+
+    pip install click==7.1.2
+    pip install transformers==4.1.1
+    pip install torch==1.7.1
+
+
+Once you have Jina installed let's take a broad overview of what we should do:
+
+.. image:: res/flow.png
+   :width: 600
+
+At the beginning of the flow, you have your data, this can be any type:
+
+* Audio
+* Image
+* Video
+* Text
+
+In this case, we are using text, so in the image, you see text only. But it can be whatever type you want.
+
+Once we have our data, as usual in Machine Learning, it's probable that you might need to pre-process that data. To keep this as simple as possible for this first tutorial, and since we are using the #TBD (dataset used TBD), we won't need to do any pre-processing. But just remember that this is a possibility for another use case.
+Once that is ready, we can encode our data into vectors and finally store those vectors, so it is ready for indexing and then querying.
+
+So what we need to do is:
```

Comment:
Nit: So the steps to build a search engine are...

Reply:
You can then have 
Step 1. Define Data 

In the next section, it just ties everything together nicely